Update styling of notifications

gustavlrsn opened the pull request:
Resolve https://github.com/opencollective/opencollective/issues/4681

# Description

### Notification bar
- [x] Updated styling
- [x] Added to Storybook

### Message box
- [x] Update styling

### Toasts
- [x] Update styling

# Screenshots

[Link to Storybook](https://opencollective-styleguide-git-feat-notifi-3d68a0-opencollective.vercel.app/)

Betree reviewed (commented):
Looking good! Happy to see design updates being reflected in the code :rocket: 

On `components/SendMoneyToCollectiveBtn.js`, it would be great to do a `addToast` (success) when the transfer completes. Currently the banner just disappear and the interface does not provide any feedback about whether it worked or not.

gustavlrsn commented on components/collective-page/CollectiveNotificationBar.js:
```diff
@@ -170,37 +201,78 @@ const getNotification = (intl, status, collective, host, LoggedInUser) => {
   }
 };
 
+export const PendingApplicationActions = ({ collective, refetch }) => {
```
Fixed!

gustavlrsn commented on components/admin-panel/sections/ActivityLog/index.js:
```diff
@@ -231,7 +231,7 @@ const ActivityLog = ({ accountSlug }) => {
         <LoadingPlaceholder width="100%" height={163} />
       ) : !data?.activities?.nodes ? (
-        <MessageBox type="error" withIcon>
+        <MessageBox type="error" withIcon title="You must be admin">
```
This was a leftover from testing having a title prop in the MessageBox, but that was removed since we already define titles in the child of the component, thanks for catching!

gustavlrsn commented:
@Betree Thanks for your review! Added toasts in `SendMoneyToCollectiveBtn.js` and fixed the other comments.

Also including a fix of the composition of the fraud protection messages. It contained a list (since there are two types of messages) which caused the layout to break a little bit.

I redid it slightly to use a common header and either be a single line or a list depending on if we show both messages or not.

One message:
<img width="1103" alt="Skärmavbild 2022-12-05 kl  13 41 52" src="https://user-images.githubusercontent.com/1552194/205641545-762ae57b-1bdd-4ad7-8601-391052461eaf.png">

Both messages:
<img width="1056" alt="Skärmavbild 2022-12-05 kl  13 41 44" src="https://user-images.githubusercontent.com/1552194/205641550-bd61ecad-2331-4944-ba6f-71a3baee88aa.png">
